src/screens/Onboarding/OnboardingItem.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  StyleSheet,
+  Text,
+  View,
+  Image,
+  useWindowDimensions,
+  ImageSourcePropType,
+} from "react-native";
+import { COLORS, FONTS, SIZES } from "../../styles/theme";
+
+interface Props {
+  id: string;
+  title: string;
+  description: string;
+  image: ImageSourcePropType;
+}
+
+const OnboardingItem: React.FC<Props> = ({ id, title, description, image }) => {
+  const { width } = useWindowDimensions();
+  return (
+    <View style={(styles.container, { width })}>
+      <Image
+        source={image}
+        style={[styles.image, { width, resizeMode: "contain" }]}
+      />
+      <View style={{ flex: 0.3 }}>
+        <Text style={styles.title}>{title}</Text>
+        <Text style={styles.description}>{description}</Text>
+      </View>
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: "#fff",
```

Comment:
@Allen-Muhani @Chegemartinx i wonder why we specify a color here. wouldnt that one be specific in the colors file?

can we have a specify a site container (or maybe several) with specific background color etc?

would that make sense?

Reply:
Yes it makes a lot of sense